server/src/db.ts

```diff
@@ -131,6 +131,12 @@ export async function addExternalIds(
   await dbConn("external_ids")
     .insert(
       toInsert.map(([system, value]: [string, string]) => {
+        if (system.includes(":")) {
+          throw new ValueError(
+            `Provider location ${id} externalIds include ${system} \
+            but ':' is not allowed`
```

Comment:
This still leaves an awkwardly large gap of several spaces between the system name and “but.” It might be worth porting `oneLine` over from the loader for this:

https://github.com/usdigitalresponse/univaf/blob/25a12f1cfb717a4bf5e57e53a0ddc8bd7d4e134c/loader/src/utils.js#L82-L108

Otherwise you could use `.replace(/\n\s*/g, " ")` instead:

```suggestion
            `Provider location ${id} externalIds include ${system}
            but ':' is not allowed`.replace(/\n\s*/g, " ")
```

(The difference with the template tag is that it doesn’t touch the content of the variables.)